ui/Modal/Org/ConfigureEns/BlockTimer.svelte

```diff
@@ -35,6 +36,19 @@
   onDestroy(() => {
     walletStore.provider.off("block", onBlock);
   });
+
+  onMount(async () => {
+    const tx = await walletStore.provider.getTransaction(txHash);
+    const block = await walletStore.provider.getBlockNumber();
+
+    // If the block has not been mined, `blockNumber` is null.
+    if (tx.blockNumber) {
+      startBlock = tx.blockNumber;
+    } else {
+      startBlock = block;
```

Comment:
If the transaction has not been included in a block yet it is not correct to use the current block as the start block. The commitment block is after the start block in this case. This leads to the UI thinking the commitment time has already passed. Here’s my suggestion how to fix this: https://github.com/radicle-dev/radicle-upstream/pull/2269